Update service-accounts-admin.md (#41156)
kubernetes/website: content/en/docs/reference/access-authn-authz/service-accounts-admin.md

1. A `serviceAccountToken` source, that contains a token that the kubelet acquires from kube-apiserver.
   The kubelet fetches time-bound tokens using the TokenRequest API. A token served for a TokenRequest expires
   either when the pod is deleted or after a defined lifespan (by default, that is 1 hour). The token

>    will be refreshed by the kubelet prior to expiry.

Reviewer: Any particular reason why you use the phrase expiry as opposed to expiration?

Author: I'm Canadian. So it's probably a British-ism I'm not aware of

Author: I don't have strong preference - if you make a inline suggestion I can accept it

>    either when the pod is deleted or after a defined lifespan (by default, that is 1 hour). The token
>    will be refreshed by the kubelet prior to expiry.

Reviewer 2: We can avoid the need to pick:
```suggestion
   either when the pod is deleted or after a defined lifespan (by default, that is 1 hour).
   The kubelet also refreshes that token before the token expires.
```

(we prefer to [use the active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice))
